xml/mobile.xml

```diff
@@ -957,30 +957,36 @@
   </variablelist>
  </sect1>
  <sect1 xml:id="sec-mobile-comm">
-  <title>Cellular Phones and PDAs</title><indexterm>
+  <title>Mobile Devices (Smartphones and Tablets)</title><indexterm>
 
   <primary>mobility</primary>
 
-  <secondary>cellular phones</secondary></indexterm><indexterm>
+  <secondary>smartphones</secondary></indexterm><indexterm>
 
   <primary>mobility</primary>
 
-  <secondary>PDAs</secondary></indexterm><indexterm>
+  <secondary>tablets</secondary></indexterm><indexterm>
 
-  <primary>cellular phones</primary></indexterm><indexterm>
+  <primary>smartphones</primary></indexterm><indexterm>
 
-  <primary>PDAs</primary></indexterm>
+  <primary>tablets</primary></indexterm>
 
   <para>
-   A desktop system or a laptop can communicate with a cellular phone via
-   Bluetooth or IrDA. Some models support both protocols and some only one of
-   the two. The usage areas for the two protocols and the corresponding
-   extended documentation has already been mentioned in
-   <xref linkend="sec-mobile-nbook-soft-wifi"/>. The configuration of these
-   protocols on the cellular phones themselves is described in their manuals.
+   A desktop system or a laptop can communicate with mobile devices via
+   Bluetooth, Wi-Fi, or a direct USB connection. Choosing a connection method
+   depends on your mobile device model and your specific needs. Connecting a
+   mobile device to a desktop machine or laptop via USB usually makes it
+   possible to work with the device as conventional external storage. Setting
+   up a Bluetooth or Wi-Fi connection allows you to interact with the mobile
+   device and control its functions directly from your desktop machine or
+   laptop. There are several open-source graphical utilities (notably <link
```

Comment:
One minor comment: If you move the parenthesis to the end of the sentence, it would be a bit easier to read and translate: 'There are several open-source graphical utilities you can use to control the connected mobile device (notably [...]).'

Reply:
Good point! Fixed. While at it, I also removed old outcommented content.